frontend/src/components/AccountReceiveTable.vue

```diff
@@ -554,14 +554,14 @@ function useReceivedFundsTable(announcements: UserAnnouncement[], spendingKeyPai
         };
 
         if (chainId === 137) {
-          // No ITX support on this network, so this is a regular transaction hash
+          // No relayer support on this network, so this is a regular transaction hash
```

Comment:
I'm not honestly sure

Reply:
@apbendi Can you share the mapping between network => relayer/tx identifier used? Once we have that we should pull this out into a separate issue so we don't lose track

Reply:
With ITX (on Mainnet/Rinkeby), their API returns a custom hash and you have to subsequently query their endpoint to figure out the list of actual network transaction hashes. That is what we pass forward from Umbra API.

With Defender, we get back the hash of the first transaction submitted, and that is what we pass forward from Umbra API. Defender [also](https://docs.openzeppelin.com/defender/relay#transaction-ids) gives you a separate, custom Identifier which can use to query their API, to find out if that first transaction has been replaced or cancelled. We currently don't expose this in any response from Umbra API.

In practice, since we're using this on Polygon and using `fast`, this virtually never happens, and so the hash we get back ends up being the hash of the transaction that is ultimately mined. This should also be the case on Optimism and Arbitrum.

If we ever started using Defender on mainnet, we might have to revisit this assumptino.

Reply:
Created https://github.com/ScopeLift/umbra-protocol/issues/329 to track this and make sure the logic in this code block is correct